deploy/packer/app/scripts/deploy.sh

```diff
@@ -47,6 +47,19 @@ docker pull taskcluster/livelog:v4
 docker pull taskcluster/dind-service:v4.0
 docker pull taskcluster/relengapi-proxy:$relengapi_proxy_version
 
+# install and configure taskcluster-worker-runner
+sudo curl --fail -L -o /usr/local/bin/start-worker https://github.com/taskcluster/taskcluster-worker-runner/releases/download/v0.2.1/start-worker-linux-amd64
```

Comment:
Would be nice to put the tc-w-runner version number in a variable in the top of the script